ap/selection/test.py

```diff
@@ -92,6 +92,17 @@ def var_HT(self: Selector, events: ak.Array, **kwargs) -> ak.Array:
     return ak.sum(jet_pt_sorted, axis=1)
 
 
+@selector(
+    uses={var_nJet, var_HT, "Jet.pt"},
+    produces={"n_jet", "ht", "jet1_pt"},
+    exposed=True,
+)
+def cutflow_features(self: Selector, events: ak.Array, **kwargs) -> None:
+    set_ak_column(events, "n_jet", self[var_nJet](events))
+    set_ak_column(events, "ht", self[var_HT](events))
+    set_ak_column(events, "jet1_pt", Route("Jet.pt[:,0]").apply(events, -999))
```

Comment:
Nice.

btw, could you use the EMPTY_FLOAT from columnar util here instead of the -999?

Reply:
good point, done